src/Symfony/Bundle/FrameworkBundle/CacheWarmer/RouterCacheWarmer.php

```diff
@@ -43,7 +43,7 @@ public function warmUp(string $cacheDir, ?string $buildDir = null): array
         $router = $this->container->get('router');
 
         if ($router instanceof WarmableInterface) {
-            return (array) $router->warmUp($cacheDir, $buildDir);
+            return $router->warmUp($cacheDir, $buildDir);
```

Comment:
The test failure for the low deps job looks related to this change.

Reply:
Actually the same problem as #57775. I added [the same line](https://github.com/symfony/symfony/pull/57814/files#diff-f9f5e8b655786315f3693fd571bc1d5f07e419805bfce50c74b7b1a82804c332R27), let's see which gets merged first :slightly_smiling_face:

Reply:
The one got merged, I rebased.